OutOfSchool/OutOfSchool.WebApi.Tests/Services/ApplicationServiceTests.cs

```diff
@@ -24,6 +24,7 @@
 using OutOfSchool.WebApi.Config;
 using OutOfSchool.WebApi.Models;
 using OutOfSchool.WebApi.Models.Application;
+using OutOfSchool.WebApi.Models.Workshop;
```

Comment:
(Redundant code) This `using` could be deleted

Reply:
And now I see that you moved it to other namespace. Previous comment is invalid, closing it